tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -569,19 +594,17 @@ bool TRTEngineOp::ExecuteTrtEngine(OpKernelContext* ctx,
 
   const bool kRetry = true;
   auto& execution_context = engine_context->execution_context;
-  const int num_binding = ctx->num_inputs() + ctx->num_outputs();
-
+  const int num_binding = cuda_engine->getNbBindings();
   std::vector<void*> buffers(num_binding);
 
   // Setup engine inputs.
   for (int i = 0; i < ctx->num_inputs(); i++) {
     const string input_name = StrCat(IONamePrefixes::kInputPHName, i);
-    const int binding_index = cuda_engine->getBindingIndex(input_name.c_str());
-    if (binding_index == -1) {
-      const string msg =
-          StrCat("Input node ", input_name, " not found, at ", name());
-      LOG(ERROR) << msg;
-      ctx->SetStatus(errors::NotFound(msg));
+    int binding_index = -1;
```

Comment:
Do we have to initialize binding_index here?

Reply:
No, it is not absolutely necessary.  I have removed the initialization.